### PR TITLE
Remove reduntant interface & type

### PR DESCRIPTION
**Pull request by ua741**
- Remove unnecessary base interface
- Remove unnecessary type interface

### CODE REVIEW

**Comment by ua741** [comment]
yarn build is failing. Checking if it's locally issue or something else.

File: apps/photos/src/types/file/index.ts [hunk 1]
```diff
@@ -5,21 +5,17 @@ import {
...
 
-interface FileAttributesBase {
-    decryptionHeader: string;
-}
-
-interface MetadataFileAttributes extends FileAttributesBase {
+export interface MetadataFileAttributes {
     encryptedData: string;
+    decryptionHeader: string;
     objectKey?: string;
```
Comment (abhinavkgrd) [inline apps/photos/src/types/file/index.ts:11]: we can remove `objectKey` property from `MetadataFileAttributes`

File: apps/photos/src/types/file/index.ts [hunk 2]
```diff
@@ -5,21 +5,17 @@ import {
...
 }
-interface S3FileAttributes extends FileAttributesBase {
+export interface S3FileAttributes {
     objectKey: string;
     encryptedData?: string;
```
Comment (abhinavkgrd) [inline apps/photos/src/types/file/index.ts:15]: and remove `encryptedData` property from `S3FileAttributes`

**Review by abhinavkgrd — APPROVED** [review]

File: apps/photos/src/types/file/index.ts (hunk 2 above)
Comment (ua741) [inline apps/photos/src/types/file/index.ts:15]: I think it's safe to remove `encryptedData` from `S3FileAttributes` and `objectKey` from `MetadataFileAttributes`. But keeping it to be on safer side for now.

Comment (ua741) [reply apps/photos/src/types/file/index.ts:15]: Okay, just saw @abhinavkgrd's comment. Will go ahead and remove

**Review by abhinavkgrd — APPROVED** [review]

**Pull request merged**